README.md

```diff
@@ -30,13 +30,21 @@ The response is a JSON with the following format:
 }
 ```
 
----
+### API key and API secret
+
+If you are an EDMdesigner intergator then you already have them.
+
+If you aren't and you would like to try or use our generator then please let us know at [support@edmdesigner.com](mailto:support@edmdesigner.com). There is currently no automatic way to pass apiKey and secret but we work on it and will announce it here as soon as it's done.
+
+### The document
+
+For EDMdesigner integrators: You can use the JSON documents fetched from EDMdesigner's API. Similarly, you will only need the "document" property from those JSONs. We are going to direct the traffic from the original generator route to this one in the future. At the moment, we support both service but we encourage our partners to try this new generator out and if it fits them replace the old one with this! We currently use it at chamaileon.io and also a few EDMdesigner integrators already use it with satisfaction.
```

Comment:
EDM api partners can use the generator in the api directly. They can send some extra settings - I sent you previously some time - and the edm api will call the new generator automagically.